SCClassLibrary/Common/Control/Volume.sc

```diff
@@ -19,31 +19,34 @@ Volume {
 			this.sendSynthDef
 		};
 
-		ServerBoot.add(initFunc)
+		ServerBoot.add(initFunc, server)
 	}
 
 	sendSynthDef {
-		server.doWhenBooted({
-			numOutputChannels = this.numChannels;
-			defName = (\volumeAmpControl ++ numOutputChannels).asSymbol;
-			SynthDef(defName, { | volumeAmp = 1, volumeLag = 0.1, gate=1, bus |
-				XOut.ar(bus,
-					Linen.kr(gate, releaseTime: 0.05, doneAction:2),
-					In.ar(bus, numOutputChannels) * Lag.kr(volumeAmp, volumeLag)
-				);
-			}).send(server);
-
-			server.sync;
-
-			updateFunc = {
-				ampSynth = nil;
-				if(persist) { this.updateSynth }
-			};
-
-			ServerTree.add(updateFunc);
-
-			this.updateSynth
-		})
+		if (server.hasBooted) {
+			fork {
+				numOutputChannels = this.numChannels;
+				defName = (\volumeAmpControl ++ numOutputChannels).asSymbol;
+				SynthDef(defName, { | volumeAmp = 1, volumeLag = 0.1, gate=1, bus |
+					XOut.ar(bus,
+						Linen.kr(gate, releaseTime: 0.05, doneAction:2),
+						In.ar(bus, numOutputChannels) * Lag.kr(volumeAmp, volumeLag)
+					);
+				}).send(server);
+
+				server.sync;
+
+				updateFunc = {
+					thisMethod.postln;
```

Comment:
Currently, the message `Volume:sendSynthDef` gets posted in the post window every time Ctrl-period is pressed. Was this line added for debugging purposes? I think we should remove it.